src/dom/document.ts

```diff
@@ -255,6 +255,10 @@ export class Document extends Node {
     return new Comment(data);
   }
 
+  createAttribute(attribute: string): Attr {
+    return new Attr(null, attribute, "", CTOR_KEY);
```

Comment:
You need to set the ownerDocument via the [`_setOwnerDocument`](https://github.com/b-fuze/deno-dom/blob/ef0bbf33d21d89ee74d3f1e65efed068852692e0/src/dom/node.ts#L153) method